manifests/agent/hiera.pp

```diff
@@ -26,49 +26,33 @@
   }
 
   $hieradata = "${codedir}/hieradata"
+  file { $hieradata:
+    ensure => link,
+    # the hieradata dir is empty so forcing to
+    # replace directory with symlink on Win 2012
```

Comment:
Does this need to be conditionalized for the Windows platform only?

Reply:
The comment is just explaining why the `force => true`, it's always been this way. All I did is remove the `if $classroom::manage_repos` conditional, because that's always true now.